examples/slash_commands/main.go

```diff
@@ -2,503 +2,77 @@ package main
 
 import (
 	"flag"
-	"fmt"
 	"log"
 	"os"
 	"os/signal"
-	"strings"
-	"time"
 
 	"github.com/bwmarrin/discordgo"
 )
 
-// Bot parameters
 var (
 	GuildID        = flag.String("guild", "", "Test guild ID. If not passed - bot registers commands globally")
 	BotToken       = flag.String("token", "", "Bot access token")
-	RemoveCommands = flag.Bool("rmcmd", true, "Remove all commands after shutdowning or not")
+	RemoveCommands = flag.Bool("rmcmd", true, "Weather to remove all commands after shutdowning or not")
 )
 
-var s *discordgo.Session
-
-func init() { flag.Parse() }
-
-func init() {
-	var err error
-	s, err = discordgo.New("Bot " + *BotToken)
-	if err != nil {
-		log.Fatalf("Invalid bot parameters: %v", err)
+func deleteCommands(ds *discordgo.Session, guildId string) {
+	for _, cmd := range commands {
+		err := ds.ApplicationCommandDelete(
+			ds.State.User.ID,
+			guildId,
+			cmd.ID,
+		)
+		if err != nil {
+			log.Fatalf("Could not delete command %s: %v\n", cmd.Name, err)
+		}
```

Comment:
Since the function is related to commands, wouldn't it be better to move it to `commands.go`?